src/App.tsx

```diff
@@ -1,61 +1,132 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
```

Comment:
You didn't add files to commit

![image](https://github.com/mate-academy/react_add-todo-form/assets/84296041/a8c9e4ff-ba59-450d-88cf-2549bb3cab81)